tox.ini

```diff
@@ -72,3 +72,27 @@ setenv =
     DB_API_URL = {env:DB_API_URL:http://localhost:8000}
 commands =
     python -m pytest tests/integration {posargs}
+
+
+[testenv:generate-server-schemas]
+skip_install = true
+deps =
+    datamodel-code-generator[http, debug]
+    ruff
+setenv =
+    DB_API_URL = {env:DB_API_URL:https://staging.openbraininstitute.org/api/entitycore/openapi.json}
+commands =
+    datamodel-codegen \
+        --url {env:DB_API_URL} \
+        --input-file-type openapi \
+        --openapi-scopes schemas \
+        --output-model-type pydantic_v2.BaseModel \
+        --target-python-version 3.11 \
+        --use-subclass-enum \
+        --use-union-operator \
+        --use-standard-collections \
+        --output {toxinidir}/src/entitysdk/_server_schemas.py \
+        --custom-template-dir {toxinidir}/templates \
+        --formatters ruff-format \
+        --use-annotated \
+        --additional-imports "enum.StrEnum"
```

Comment:
It can be worth to mention the purpose of the new tox command in the README in the dev section.
I see that the generation is used for types, but it can be useful also to spot changes in the server schema, although it isn't used to automatically update the client schemas.
We could also think about integrating it in the CI, although it can be debatable if it should fail if the schema isn't up to date with the openapi spec in staging.